ex-2-selection-sort/selection-sort.js

```diff
@@ -0,0 +1,31 @@
+function findSmallest( arr ){
+    let smallest = arr[0];
+    let smallestIndex = 0;
+
+    for (let i = 0; i < arr.length; i++) {
+        if( arr[i] < smallest ){
+            smallest = arr[i];
+            smallestIndex = i;
+        }
+    }
+
+    return smallestIndex;
+
+}
+
+function selectionSort( arr ){
+    let sorted = [];
```

Comment:
If you had to choose between a linked list and an array for this variable, which would you choose? Why?

Reply:
For the sort itself, I think a linked list would be better, given that inserts would be quicker.  I think it would ultimately depend on the end use of the list.  If the list will need random access, like a search, an array would be better.  If the list is something like a queue or stack that only needs adjacent access, a linked list would definitely be the way to go.

Reply:
Right on!!